### PR TITLE
Configure deployment to Maven Central via Sonatype

### LINKED ISSUE #9: Deploy to Maven Central
I know this is a bit complicated but I'd really appreciate if you'd deployed this to Maven Central.

I've done it a few times before and can help you with it.

### PR DESCRIPTION
**Pull request by leonardehrenfried**
Resolves #9.

You need to configure your Sonatype credentials like this:

Create a file `$HOME/.sbt/(sbt-version)/sonatype.sbt` with the following content:
```
credentials += Credentials("Sonatype Nexus Repository Manager",
        "oss.sonatype.org",
        "(Sonatype user name)",
        "(Sonatype password)")
```
Then either run `sbt publishSigned` (which will deploy a snapshot) or `sbt release`.

### CODE REVIEW

**Comment by leonardehrenfried** [comment]
This is causing travis to fail: 
```
/home/travis/.travis/job_stages: line 53: /usr/local/bin/sbt: Permission denied
```
Not sure what's going on but appears to be a problem with Travis itself.

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,10 +1,7 @@
-resolvers += Resolver.url(
-  "bintray-sbt-plugin-releases",
-    url("http://dl.bintray.com/content/sbt/sbt-plugin-releases"))(
-        Resolver.ivyStylePatterns)
+addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.1.12")
 
-addSbtPlugin("me.lessis" % "bintray-sbt" % "0.1.1")
+addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.5")
 
-addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.1.12")
```
Comment (velvia) [inline project/plugins.sbt]: Why remove sbt-jmh?

**Comment by velvia** [comment]
I think this looks fine.  I'm not sure why the travis is failing.

**Comment by xuwei-k** [comment]
https://github.com/travis-ci/travis-ci/issues/7703

**Comment by velvia** [comment]
@xuwei-k thanks.

@leonardehrenfried do you want to try the travis workaround mentioned above?

For now, please add:

    before_script:
      - sudo chmod +x /usr/local/bin/sbt

Thanks.

**Comment by leonardehrenfried** [comment]
Yes, will try it out.

**Pull request merged**